src/styles/fonts.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(../fonts/Roboto-Regular.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(../fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(../fonts/Roboto-Bold.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+.page {
+  font-family: "Roboto" , Arial, sans-serif;
```

Comment:
move it to the html selector

Reply:
.page is  class of html tag
in video examples said that according to BEM we shouldn't use global selectors and selector by tag